src/BBCodeParser.php

```diff
@@ -24,7 +24,7 @@ class BBCodeParser extends TextParser {
 	 * @config
 	 * @var Boolean Set whether similies :), :(, :P are converted to images
 	 */
-	private static $allow_similies = false;
+	private static $allow_smilies = false;
```

Comment:
Hmm, where? I see it set [correctly](https://github.com/zanderwar/silverstripe-bbcodeparser/blob/1755da9fadfbf556fa7d06da8ebad8d8c075f836/src/BBCodeParser.php#L36)